package.json

```diff
@@ -4,30 +4,32 @@
   "description": "Minard backend",
   "main": "server.js",
   "dependencies": {
-    "@reactivex/rxjs": "^5.0.1",
-    "aws-sdk": "^2.7.13",
-    "bluebird": "^3.4.6",
+    "@reactivex/rxjs": "^5.0.3",
+    "aws-sdk": "^2.7.28",
+    "bluebird": "^3.4.7",
     "boom": "^4.2.0",
-    "cache-manager": "^2.2.0",
-    "cache-manager-redis": "^0.3.2",
+    "cache-manager": "^2.4.0",
+    "cache-manager-redis": "^0.4.0",
     "datauri": "^1.0.5",
     "deepcopy": "^0.6.3",
-    "eventstore": "1.9.0",
-    "extract-zip": "^1.5.0",
+    "eventstore": "^1.11.1",
+    "extract-zip": "^1.6.0",
     "form-data": "^2.1.2",
     "good": "^7.1.0",
     "good-console": "^6.2.0",
     "good-squeeze": "^5.0.1",
     "gravatar": "^1.6.0",
     "h2o2": "^5.4.0",
-    "hapi": "^15.2.0",
-    "hapi-raven": "^4.3.1",
-    "inert": "^4.0.3",
-    "inversify": "^2.0.1",
-    "joi": "^10.0.5",
+    "hapi": "^16.1.0",
+    "hapi-auth-jwt2": "^7.2.4",
```

Comment:
Nevermind, I now remembered that this PR does, in fact, now have much more than just updated packages in it.